lib/http-conditional.php

```diff
@@ -182,7 +182,7 @@ function httpConditional(int $UnixTimeStamp, int $cacheSeconds = 0, int $cachePr
  * Reference rfc2616-sec14.html#sec14.11
  */
 function _httpConditionalCallBack(string $buffer, int $mode = 5): string {
-	if (extension_loaded('zlib') && (!ini_get('zlib.output_compression'))) {
+	if (extension_loaded('zlib') && (ini_get('zlib.output_compression') === false)) {
```

Comment:
I think this is wrong as well:
* https://php.net/function.ini-get
* https://php.net/zlib.configuration#ini.zlib.output-compression

```suggestion
	if (extension_loaded('zlib') && (ini_get('zlib.output_compression') == false)) {
```